tc_aws/aws/bucket.py

```diff
@@ -35,9 +38,10 @@ def get(self, path, callback=None):
         :param string path: Path or 'key' to retrieve AWS object
         :param callable callback: Callback function for once the retrieval is done
         """
-        my_session = session_handler.get_session()
+        my_session = session_handler.get_session(self._endpoint)
```

Comment:
I'd rather see a boolean value than a string one here, if you can make the change